docs/ilm.asciidoc

```diff
@@ -59,6 +60,40 @@ apm-server:
 
 ILM can now be customized via the remaining `apm-server.ilm.setup.*` configuration options.
 
+[float]
+[[custom-ilm-alias]]
+==== Create a custom ILM rollover_alias
+
+You can define a custom `rollover_alias` per event type.
```

Comment:
We should probably also explain what a rollover alias is.

Reply:
Good point, added some description.